modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1138 @@
+
+The consequence of the previous UX, meant that adjustments needed to be made to the horizontal lines within the *help menu* table, that seperate each `row`.
+
+To fix this accessibility issue, {productname} 6.4.1, increased the contrast ratio to `3:1` for each line divider for better visibility
+
+As a result, the `help menu` is now styled to support a wider range of users, by these new changes implemented.
+
+=== The sidebar element now has accessibility role region when visible and presentation when hidden
+//#TINY-9517
+
+In previous versions of {productname}, Axe-DevTools (web accessibility testing extension) identified that the sidebar element within {productname} was set to `role="complementary"` even when the element was hidden. This created issues with screen readers, as the general rule is that landmarks should not be contained within another landmark.
+
+As a consequence by having the sidebar set to `role="complementary"`, tools such as screen readers in some situations would disable their functionality.
+
+To fix this issue, changes were made to the role attribute of the sidebar element, which is now set to `role="region"` when it's active state is visible, otherwise it defaults to `role="presentation"` when hidden.
+
+This is important as the role of `region` ensures the sidebar and its contents are accessible to screen readers and other accessibility tools. When the sidebar element is hidden, it has a role of `presentation`, which is used to inform accessibility tools that the element can be safely ignored. 
+
+In {productname} 6.4.1, when the sidebar is either visible or hidden it now passes all the identified accessibility requirements.
+
+=== Updated the Code Sample and Template plugin dropdown menu styles to match the default {productname} styles
+//#TINY-9630
+
+Previously, the Code Sample and Template open source plugins used browser-native elements for some dropdown menus.
+
+With this update, these menus are now created using the equivalent {productname} custom element, making these dropdown menus present with the same styling as other {productname} UI elements.
+
+
+[[bug-fixes]]
+== Bug fixes
+
+{productname} 6.4.1 also includes the following bug fixes:
+
+=== Sometimes the editor would finish initializing before the theme would have finished loading
+//#TINY-9556
+
+In earlier versions of {productname}, the properties set for the `renderUI` could cause conflicts with the theme load times. As a consequence, the `renderUI` would finish rendering before the {productname} theme it relied upon was initialized.
+
+To fix this bug, {productname}, made changes to the `renderUI` properties that allowed the theme to return a Promise, resolving the load ordering issues. 
+
+In {productname} 6.4.1, all `renderUI` property loading issues have been resolved.
+
```

Comment:
I missed this when I corrected the other copy of it under additions (search for TINY-9556). We don't need two.

```suggestion
```